src/ex2/PokemonClient.js

```diff
@@ -0,0 +1,11 @@
+export class PokemonClient {
+  constructor() {
+    this.api = "https://pokeapi.co/api/v2/pokemon/"
+  }
+
+  async getPokemon(name) {
+    const response = await fetch(`${this.api}${name}`)
+    const json = await response.json()
+    return json
+  }
+}
```

Comment:
it make sense to have here also get multiple pokemons with the promise all and everything, currently you did that in app.js and if you'll want to re-use it you won't be able to